uint32 device_id	# Sensor ID that must be unique for each baro sensor and must not change

uint64 error_count

float32 pressure	# static pressure measurement in millibar

float32 temperature	# static temperature measurement in deg C
